doc/qbittorrent-nox.1

```diff
@@ -1,47 +1,240 @@
+\f[I]OPTIONS\f[R] can be passed in from several sources.
+If the same option is specified in multiple sources, the last one takes
+precedence, in the following order:
+.IP "1." 3
+The main configuration file (see the \f[I]FILES\f[R] section)
+.IP "2." 3
+\f[I]ENVIRONMENT\f[R] variables
+.IP "3." 3
+Command-line \f[I]OPTIONS\f[R]
+.PP
+\f[B]IMPORTANT NOTE:\f[R] Settings changed via command-line
+\f[I]OPTIONS\f[R] or \f[I]ENVIRONMENT\f[R] variables not only
+\f[B]override\f[R] their corresponding values in the configuration file,
+they \f[B]also overwrite\f[R] them.
+.PP
+\f[I]ENVIRONMENT\f[R] variables and command-line \f[I]OPTIONS\f[R] can
+currently only override a small subset of all the available settings
+that can be configured in the main configuration file.
+.PP
+qBittorrent-nox can be cleanly terminated by sending the
+\f[B]SIGINT\f[R] or \f[B]SIGTERM\f[R] signals to its process.
 .SH OPTIONS
 .PP
-\f[B]\f[CB]--help\f[B]\f[R] Prints the command line options.
+TODO: options: - no torrent creator - no skip splash option - daemon:
+start in background, cant use if already started (but probably instances
+with different configuration names can still coexist)
+.PP
+TODO: document the fact that the main config file is populated upon
+first save.
+qbittorrent-conf manpage does not exist, but it\[cq]s better to just
+tell users to start once to generate it than to create and maintain such
+a man page
+.SH ENVIRONMENT
+.PP
+\f[I]OPTIONS\f[R] may be supplied via environment variables with
+equivalent names.
+.PP
+For an option named \f[B]parameter-name\f[R], the corresponding
+environment variable name is \f[B]QBT_PARAMETER_NAME\f[R].
+In other words, the parameter name is converted to upper-case, any
+hyphens (\f[B]-\f[R]) are replaced by underscores (\f[B]_\f[R]), and
+\f[B]QBT_\f[R] is prepended to it.
 .PP
-\f[B]\f[CB]--version\f[B]\f[R] Prints qbittorrent program version
-number.
+To pass flag values, set the variable to \f[B]1\f[R] or \f[B]TRUE\f[R].
+For example, \f[B]\f[CB]--add-paused=true\f[B]\f[R] becomes
+\f[B]QBT_ADD_PAUSED=1\f[R]
+.SH FILES
+.SS Standard configuration files directory hierarchy
 .PP
-\f[B]\f[CB]--webui-port=x\f[B]\f[R] Changes Web UI port to x (default:
-8080).
+Currently, qBittorrent complies somewhat with the Freedesktop XDG Base
+Directory specification.
+The configuration files are stored in the default directories of the
+specification:
+.IP "1." 3
+Configuration files are stored in \f[I]\[ti]/.config/qBittorrent\f[R]
+.IP "2." 3
+Data files are stored in \f[I]\[ti]/.local/share/data/qBittorrent\f[R]
+.IP "3." 3
+Cache is stored in \f[I]\[ti]/.cache/qBittorrent\f[R]
+.PP
+However, qBittorrent will not use custom XDG paths set via the
+\f[B]$XDG_CONFIG_HOME\f[R] environment variables and the like; it will
+always use the default ones.
+.PP
+The \f[B]\f[CB]--configuration\f[B]\f[R] and
+\f[B]\f[CB]profile\f[B]\f[R] options will instruct qBittorrent to
+instead create/use configuration files in different places.
```

Comment:
"to instead create ..." sounds wrong for me. Although my English is bad...

Reply:
google gives plenty of results for "to instead create" so i think it's legit.

Reply:
I think it can confuse some users a bit. How about =

> options will **instead** instruct qBittorrent to create/use configuration files in different places.

Reply:
Thanks for the suggestions, I went with `options will instruct qBittorrent to create/use configuration files in different locations, instead of the default ones.` in the end.